build_all.sh

```diff
@@ -17,6 +17,24 @@ set -o pipefail
 # shellcheck source=common_functions.sh
 source ./common_functions.sh
 
+# summary table array
+summary_table_file="./.summary_table"
+
+function create_summary_table_file() {
+	touch ${summary_table_file}
+	echo "+------------------------------------------------------------------------------+----------+" >> ${summary_table_file}
+	echo "|                                 Docker image                                 |  Status  |" >> ${summary_table_file}
+	echo "+------------------------------------------------------------------------------+----------+" >> ${summary_table_file}
+}
+
+function print_summary_table() {
+	cat ${summary_table_file}
+}
+
+function remove_summary_table_file() {
+	rm -rf ${summary_table_file}
```

Comment:
I'd suggest `rm -f` as it is just a file

Reply:
Aah! I shouldn't have added a recursive option. Changed it to `rm -f`